scripts/example.coffee

```diff
@@ -1,108 +0,0 @@
-# Description:
-#   Example scripts for you to examine and try out.
-#
-# Notes:
-#   They are commented out by default, because most of them are pretty silly and
-#   wouldn't be useful and amusing enough for day to day huboting.
-#   Uncomment the ones you want to try and experiment with.
-#
-#   These are from the scripting documentation: https://github.com/github/hubot/blob/master/docs/scripting.md
-
-module.exports = (robot) ->
-  robot.respond /repo/i, (res) ->
```

Comment:
Given that removing this also removes `!repo`, and there's no harm in having some trivial default examples, I'd rather keep this one or at the very least, replace it with a script just for `!repo`?

Reply:
Ah totally missed the fact this has !repo, will write one now :)

As for example scripts, I agree with the conversation on slack in that the repo already contains many great and simple examples for writing scripts, so I don't think a dedicated example file is necessary, but maybe that's something we can look to building in future :)

Reply:
Done